libs/solid/profile/src/lib/services/profile-definition.service.ts

```diff
@@ -25,21 +26,183 @@ export class ProfileDefinitionService {
     @Inject(SOLID_CORE_CONFIG) private _config: SolidCoreConfig
   ) {}
 
+  //Now i created 2 ways to calling OpenAPI, one is for the 2.0 Version,
+  //another is for the 3.0 Version. Just to make sure the other APPs
+  //work as well, which haven't had the OpenAPI 3.0 Version yet.
+
+  //OpenAPI 3.0
   public loadDefinitions() {
+    //prevent AIS, Dive, Planty calling OpenAPI 3.0
+    //so we don't have duplicated data in profile
+    if (
+      this._config.appName === 'AIS' ||
+      this._config.appName === 'Div-e' ||
+      this._config.appName === 'PLANTY2Learn'
+    ) {
+      return;
+    }
+    return this.http.get<OpenApi>(`${this._config.apiUrl}/api/schema`).pipe(
+      map((swagger) => {
```

Comment:
Bleibt nur noch die Frage, ob wir die `swagger`-Variable umbenennen (z.B. in `openapi` oder `openapi_object`)